src/components/browse_daos/dao_card.tsx

```diff
@@ -14,12 +19,41 @@ type DaoCardProps = {
 	chainId: SupportedChainId | undefined
 	noOfApplicants: number
 	totalAmount: number
+	workspaceData: any
 }
 
-function DaoCard({ logo, isAdmin, name, daoId, chainId, noOfApplicants, totalAmount, onVisibilityUpdate, isVisible }: DaoCardProps) {
+function DaoCard({ logo, isAdmin, name, daoId, chainId, noOfApplicants, totalAmount, onVisibilityUpdate, isVisible, workspaceData }: DaoCardProps) {
 	const router = useRouter()
 	const [isActive, setIsActive] = useState(false)
 	const { t } = useTranslation()
+	const [totalFundsDisbursed, setTotalFundsDisbursed] = useState<number>(0)
+	const { subgraphClients } = useContext(ApiClientsContext)!
+
+
+	const { data: fundTransfersData } = useGetAllFundsTransfersForADaoQuery({
+		client:
+			subgraphClients[
+				getSupportedChainIdFromSupportedNetwork(workspaceData?.supportedNetworks[0]!) || defaultChainId
+			].client,
+		variables: {
+			workspaceId: daoId || '',
+		},
+	})
+
+	useEffect(() => {
+		if(fundTransfersData) {
+			let totalAmount = 0
+			for(let i = 0; i < fundTransfersData?.fundsTransfers?.length!; i++) {
+				if(fundTransfersData?.fundsTransfers[i]?.status === 'executed'
+						&& fundTransfersData?.fundsTransfers[i]?.type === 'funds_disbursed_from_safe') {
```

Comment:
This accounts for fund transfers that were initiated from the safe. We need to take into consideration the fund transfers of type "fund_disbursed" as well.

Reply:
@richikchanda1999 there are 2 parts, one which is coming from the props, and other which is only calculated from SAFE. 
Adding and showing the sum of them.